build/dockerfiles/rhel.Dockerfile

```diff
@@ -20,15 +21,21 @@ RUN chmod +x /usr/share/container-scripts/httpd/pre-init/40-ssl-certs.sh && \
 RUN yum update -y systemd && yum clean all && rm -rf /var/cache/yum && \
     echo "Installed Packages" && rpm -qa | sort -V && echo "End Of Installed Packages"
 
-# BEGIN these steps might not be required
+RUN echo "<FilesMatch "\""^\\.ht"\"">" >> /etc/httpd/conf/httpd.conf && \
```

Comment:
If I look at the current file
```
$ docker run --rm -it --entrypoint=sh registry.access.redhat.com/ubi8/httpd-24:1-143 -c "cat /etc/httpd/conf/httpd.conf"
```

 it contains

```#
# The following lines prevent .htaccess and .htpasswd files from being
# viewed by Web clients.
#
<Files ".ht*">
    Require all denied
</Files>
```

Reply:
@benoitf Thank you. I will remove this section.

Reply:
Removed